tests/platform_tests/api/test_chassis.py

```diff
@@ -381,3 +381,15 @@ def test_get_watchdog(self, duthost, localhost, platform_api_conn):
     def test_get_eeprom(self, duthost, localhost, platform_api_conn):
         eeprom = chassis.get_eeprom(platform_api_conn)
         pytest_assert(eeprom is not None, "Failed to retrieve system EEPROM")
+
```

Comment:
Suggest having a separate PR for the platform API test.

Reply:
For this PR, all test cases were checked in at the first commit. In order to seperate API test and SNMP test, I will create two branch manually. Will let you know once it is finished.